apps/api_web/config/config.exs

```diff
@@ -30,6 +30,26 @@ config :api_web, :rate_limiter,
   max_registered_per_interval: 100_000,
   wait_time_ms: 0
 
+config :api_web, :rate_limiter_concurrent,
+  enabled: false,
+  memcache: false,
+  # How many seconds tolerated when calculating whether a connection is still open
+  # 45 - 30 (see ApiWeb.EventStream.Initialize's timeout value) gives us a buffer of 15 seconds:
+  heartbeat_tolerance: 45,
+  connection_opts: [
+    namespace: "api_concurrent_rate_limit_dev",
+    # The total lifetime (s) of an API key + request type (static or streaming) key:
+    ttl: 60,
+    # Allows us to hold a nested payload per key, this is necessary because memcached does not offer
+    # a great way to group keys:
+    coder: Memcache.Coder.JSON
+  ],
+  # Default concurrent connections - these can be overridden on a per-key basis in the admin UI:
+  max_anon_streaming: 1,
```

Comment:
suggestion: I forget if I added this as a comment before, but there are no anonymous streams. An API key is required to use the event-stream interface.

Reply:
You're right, I don't think this code path is reachable. Should I remove it? It doesn't *harm* anything but I could see the value in having less code if we don't think we will ever allow anonymous streams.

Reply:
All set.